subsys/llext/shell.c

```diff
@@ -182,10 +192,46 @@ static int cmd_llext_call_fn(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+#ifdef CONFIG_LLEXT_FILE_SYSTEM
+static int cmd_llext_load_fs(const struct shell *sh, size_t argc, char *argv[])
+{
+	int res;
+	struct fs_dirent dirent;
+
+	res = fs_stat(argv[2], &dirent);
+	if (res) {
+		shell_error(sh, "Failed to obtain file %s, return code %d\n", argv[2], res);
+		return -ENOEXEC;
+	}
+
+	if (dirent.type != FS_DIR_ENTRY_FILE) {
+		shell_error(sh, "Not a file %s", argv[2]);
+		return -ENOEXEC;
+	}
+
+	struct llext_fs_loader fs_loader = LLEXT_FS_LOADER(argv[2]);
+	struct llext_loader *ldr = &fs_loader.loader;
+	struct llext_load_param ldr_parm = LLEXT_LOAD_PARAM_DEFAULT;
+	struct llext *ext;
+
+	res = llext_load(ldr, argv[1], &ext, &ldr_parm);
+	if (res == 0) {
+		shell_print(sh, "Successfully loaded extension %s, addr %p\n", ext->name, ext);
+	} else {
+		shell_print(sh, "Failed to load extension %s, return code %d\n", ext->name, res);
+		return -ENOEXEC;
```

Comment:
same with the error code? If you end up updating the PR, maybe use
```
if (res < 0) {
    shell_print(...);
    retirm res:
}
shell_print(...);
return 0;
```
similar in `llext_fs_seek()`